### PR TITLE
User is able to edit comments

### PR DESCRIPTION
**Pull request by Hemesh-Unka**
Tests are passing and delete method is in place.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,4 +9,5 @@
     resources :likes
   end
 
+  resources :comments
```
Comment (rewitt94) [inline config/routes.rb]: This has been changed in Likes Two also. Will this create merge conflicts?

**Review by rewitt94 — APPROVED** [review]
Good stuff.

**Review by defc0nONE — APPROVED** [review]
Using best_in_place as comments, no-ice.

File: config/routes.rb (hunk 1 above)
Comment (Hemesh-Unka) [reply config/routes.rb]: We shall wait and see...

**Pull request merged**